lib/rules/web/ocm_console/cluster_list_page.xyaml

```diff
@@ -157,50 +142,50 @@ cluster_list_table_columns:
         xpath: //table/thead//th[text()='Provider (Location)']
     - selector:
         xpath: //table/thead//th[@data-key='6']
-switch_to_cluster_detail_page:
+go_to_cluster_detail_page:
     element:
         selector:
             xpath: //table[@class='pf-c-table pf-m-grid-md']/tbody/tr/td/a[contains(@href,'/openshift/details/<cluster_id>')]
         op: click
-expand_cluster_list_page_actions_menu:
+expand_cluster_actions_on_cluster_list_page:
     element:
         selector:
             xpath: //table[@class='pf-c-table pf-m-grid-md']/tbody/tr/td/a[contains(@href,'/openshift/details/<cluster_id>')]/../..//button[@aria-label='Actions']
         op: click
-close_cluster_list_page_actions_menu:
-    action: expand_cluster_list_page_actions_menu
-actions_menu_all_enabled:
-    action: dropdown_launch_console_enabled
-    action: scale_cluster_enabled
-    action: delete_cluster_button_enabled
-    action: edit_display_name_enabled
-list_action_partial_enabled:
-    action: dropdown_launch_console_disabled
-    action: scale_cluster_button_disabled
-    action: delete_cluster_button_enabled
-    action: edit_display_name_enabled
+close_cluster_actions_on_cluster_list_page:
+    action: expand_cluster_actions_on_cluster_list_page
+check_cluster_actions_on_cluster_list_page_all_enabled:
+    action: enabled_launch_console_button_on_cluster_list_page_loaded
+    action: enabled_scale_cluster_button_on_cluster_list_page_loaded
+    action: enabled_delete_cluster_button_on_cluster_list_page_loaded
+    action: enabled_edit_display_name_button_on_cluster_list_page_loaded
+check_cluster_actions_on_cluster_list_page_partial_enabled:
+    action: disabled_launch_console_button_on_cluster_list_page_loaded
+    action: disabled_scale_cluster_button_on_cluster_list_page_loaded
+    action: enabled_delete_cluster_button_on_cluster_list_page_loaded
+    action: enabled_edit_display_name_button_on_cluster_list_page_loaded
 
-click_actions_launch_console_button:
+click_launch_console_button_on_cluster_list_page:
     element:
         selector:
             xpath: //*[@class='pf-c-dropdown__menu-item' and text()='Launch Console']
```

Comment:
I do not think this rule is an individual xpath if there are multi clusters. The below rules on the cluster list page have the same issue.